tests/fixtures/negative-pgid-kill-audit.c

```
@@ -0,0 +1,188 @@
+  return false;
+}
+
+static void remember_absent_group(pid_t target) {
+  if (group_was_absent(target) ||
+      absent_group_count == MAX_ABSENT_IDENTITIES) {
+    return;
+  }
+  absent_groups[absent_group_count++] = target;
+}
+
+void negative_pgid_kill_audit_remember_absent(pid_t target) {
```

Cppcheck (2.21.0)

[style] The function 'negative_pgid_kill_audit_remember_absent' is never used.

(unusedFunction)